### PR TITLE
Added model schema and migrations

### PR DESCRIPTION
**Pull request by IshmeetSingh06**

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -17,6 +17,8 @@
 default: &default
   adapter: postgresql
   encoding: unicode
+  host: localhost
+  port: 5000
```
Comment (gagan93jtg) [inline config/database.yml]: why the specific port?

File: test/models/user_test.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
```
Comment (gagan93jtg) [inline test/models/user_test.rb]: remove fixtures and test files. We prefer Rspec with factories. Both of which will be added later. For now, skip committing test files

**Review by gagan93jtg — DISMISSED** [review]

File: db/migrate/20230821082821_create_events.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+class CreateEvents < ActiveRecord::Migration[7.0]
+  def change
+    create_table :events do |t|
+      t.string :name
+      t.text :description
+      t.string :location
+      t.datetime :date_time
```
Comment (PramanSatya) [inline db/migrate/20230821082821_create_events.rb:7]: date_time -> time

File: db/migrate/20230821082856_create_registrations.rb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+class CreateRegistrations < ActiveRecord::Migration[7.0]
+  def change
+    create_table :registrations do |t|
+      t.references :user, null: false, foreign_key: true
```
Comment (PramanSatya) [inline db/migrate/20230821082856_create_registrations.rb:4]: AFAIK we don't need ```null: false``` here
Please check there is already a validation in rails to check null for associations unless ```optional: true``` is given

File: app/models/user.rb [hunk 5]
```diff
@@ -0,0 +1,2 @@
+class User < ApplicationRecord
```
Comment (PramanSatya) [inline app/models/user.rb:1]: user can have many registrations, right?
Also, admin user can have many events

**Review by PramanSatya — CHANGES_REQUESTED** [review]

**Review by PramanSatya — APPROVED** [review]

**Pull request merged**